guides/image-sources/provisioning-installer-bootdisk-subnet.plantuml

```diff
@@ -0,0 +1,59 @@
+@startuml
+
+!include foreman.pstyle
+!$networkboot = 1
+
+title Installer-based provisioning with subnet bootdisk on bare metal
+
+actor       User
+participant "Provisioned\nHost" as Host
+participant "Foreman" as Foreman
+participant "Foreman\nProxy" as Proxy
+'participant TFTP
+participant DHCP
+participant DNS
+!if ($puppet)
+participant "Puppet\nserver" as Puppet
+!endif
+
```

Comment:
I don't see the `== Create host in Foreman ==` section describing details of the iso generation. #3308

Reply:
Because the user isn't required to create a host for a subnet boot disk AFAIK.

Reply:
That's true, but the image is useless without a created host.
If you boot a VM from the ISO and the related host is not created in Foreman, the provisioning process will fail.